src/components/map/MapService.js

```diff
@@ -495,6 +510,20 @@ goog.require('ga_urlutils_service');
                 }
               });
 
+              var shop = [
+                'ch.swisstopo.pixelkarte-farbe-pk25.noscale',
+                'ch.swisstopo.pixelkarte-farbe-pk50.noscale',
+                'ch.swisstopo.pixelkarte-farbe-pk100.noscale',
+                'ch.swisstopo.pixelkarte-farbe-pk200.noscale',
+                'ch.swisstopo.swissimage-product',
+                'ch.swisstopo.swissbuildingds3d'
+              ];
+              shop.forEach(function(id) {
+                if (response.data[id]) {
+                  response.data[id].tooltip = true;
+                  response.data[id].available = true;
+                }
+              });
```

Comment:
@AFoletti @rfrancel Can we put this in bod so we can remove it from front-end code? 

Please do so and deploy bod to dev.